diagnostic.md

```diff
@@ -5,74 +5,91 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+to organize the data/processes needed for the web application to communicate between the server and client
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Serializers replace all the views we care about
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+CREATE
+READ
+UPDATE
+DESTROY
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+.exists?
```

Comment:
Looking for `Index (READ), Create (CREATE), Show (READ), Update (UPDATE), Destroy (DESTROY).`